app/_src/gateway/get-started/ai-gateway.md

```diff
@@ -0,0 +1,192 @@
+
+However, to help you get started quickly, we have provided a script that automates the task of 
+deploying a {{site.base_gateway}} configured as an AI Gateway to a Docker container, 
+allowing you to evaluate the core AI provider capabilities before exploring the full suite of AI plugins.
+
+The quickstart script supports deploying {{site.base_gateway}} on {{site.konnect_product_name}} or 
+in traditional mode on Docker only.
+
+{:.note}
+> **Note:**
+> Running this script prompts you for AI Provider API Keys which are used to configure authentication with
+> hosted AI providers. These keys are passed to the {{site.base_gateway}} Docker container and are 
+> not transmitted outside the host machine.
+
+The script runs interactively and prompts you for information about the AI providers you wish to configure.
+
+Download and run script shell in one step:
+
+```sh
+curl -Ls https://get.konghq.com/ai | bash
+```
+
+Once the AI Gateway is deployed, you will see some information displayed to help you evaluate the AI provider 
+proxy behavior. For example, the script will output a sample command helping you route a chat request
+to OpenAI:
+
+```sh
+=======================================================
+ ⚒️                                 Routing LLM Requests
+=======================================================
+
+Your AI Gateway is ready. You can now route AI requests
+to the configured AI providers. For example to route a
+chat request to OpenAI you can use the following
+curl command:
+
+curl -s -X POST localhost:8000/openai/chat \
+ -H "Content-Type: application/json" -d '{
+   "messages": [{
+   "role": "user",
+   "content": "What is Kong Gateway?"
+ }] }'
+```
+
+Additionally, the script will display information about deployment
+files it generates which can be used to for future AI Gateway configurations. 
+
+```sh
+=======================================================
+ ⚒️                What is next with the Kong AI Gateway
+=======================================================
+
+This script demonstrated the installation and usage
+of only one of the many AI plugins that Kong Gateway
+provides (the 'ai-proxy' plugin).
+
+See the output directory to reference the files
+used during the installation process and modify for
+your production deployment.
+ℹ /tmp/kong/ai-gateway
+```
+
+## AI Gateway Capabilities 
+
+The following describes the broad capabilities of the AI Gateway. More details can be found
+in the AI Gateway plugins found in the {{site.base_gateway}} [Plugin Hub](/hub/?category=ai).
+
+### AI Provider Proxy
+
+The core of the AI Gateway is the ability to route AI requests to various providers exposed via 
+a provider agnostic API. This normalized API layer affords organizations and their developers 
+multiple benefits:
+
+* Client applications are shielded from AI provider API specifics promoting code reusability
```

Comment:
```suggestion
* Client applications are shielded from AI provider API specifics, promoting code reusability
```

Reply:
The word "reusability" just needs to be added to the vale dictionary: https://github.com/Kong/docs.konghq.com/blob/feat/ai-plugins-get-started/.github/styles/kong/dictionary.txt